internal/consumer/cron.go

```diff
@@ -0,0 +1,35 @@
+package consumer
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/emmaLP/gs-software-onboarding/internal/model"
+	"github.com/robfig/cron/v3"
+	"go.uber.org/zap"
+)
+
+func ConfigureCron(ctx context.Context, logger *zap.Logger, config *model.Configuration) error {
+	c := cron.New()
+
+	var err error
+	service, err := NewService(logger, &config.Consumer, nil)
+	if err != nil {
+		return fmt.Errorf("An error occurred when trying to instantiate the consumer service: %w", err)
+	}
+	storyProcessing := func() {
+		err = service.processStories(ctx)
+	}
+	storyProcessing()
```

Comment:
To make this a bit clearer is it worth calling `service.processStories(ctx)` directly for the initial call?

i.e.

```
if err = service.processStories(ctx); err != nil {
    return fmt.Errorf("Error occurred processing stories: %w", err)
}

_, err = c.AddFunc(config.Consumer.CronSchedule, func() {
    service.processStories(ctx)
})

```